app/portainer/hostmanagement/fdo/fdo.service.ts

```diff
@@ -23,13 +23,66 @@ export async function configureDevice(
   }
 }
 
+export async function createProfile(
+  profileName: string,
+  method: string,
+  profileFileContent: string
+) {
+  const payload: Record<string, unknown> = {
+    Name: profileName,
```

Comment:
you can actually use regular camelCase. go server doesn't care about casing

Reply:
done